external/runtime/runtime.depproj

```diff
@@ -73,6 +75,43 @@
     <Exec Command="chmod +x $(TestHostRootPath)dotnet" Condition="'$(RunningOnUnix)' == 'true'"/>
   </Target>
 
+  <PropertyGroup>
+    <_CoreCLRPackageId>runtime.$(NugetRuntimeIdentifier).Microsoft.NETCore.Runtime.CoreCLR</_CoreCLRPackageId>
+    <_CoreCLRSymbolPackageUrl>https://dotnet.myget.org/F/dotnet-core/symbols/$(_CoreCLRPackageId)/$(CoreClrPackageVersion)</_CoreCLRSymbolPackageUrl>
```

Comment:
Note: The path to get these symbols packages may be changing in the near future. You can chat with me offline if you want more details.

cc @markwilkie @karajas